qa/pom.xml

```diff
@@ -348,6 +348,17 @@
           <artifactId>openjdk-orb</artifactId>
           <version>${version.org.jboss.openjdk-orb}</version>
         </dependency>
+        <dependency>
+          <groupId>com.sun.corba</groupId>
+          <artifactId>glassfish-corba-omgapi</artifactId>
+          <version>3.2.0-b002</version>
```

Comment:
I think this version should go into the root pom.xml (same for the next added dependency)

Reply:
okay